### ISSUE TITLE
Topo expansível e imagem de Fundo do cabeçalho não aparece. Manual indica link incorreto para configurar a funcionalidade.

### ISSUE THREAD

**Issue by idgserpro** [plonegovbr/brasil.gov.portal#574]
**Qual versão do produto você está usando e em qual sistema operacional?**

Portal Padrão 2.1.1
Plone 4.3.18 (4320)
CMF 2.2.10
Zope 2.13.28

**Possui plugins adicionados além dos definidos numa instalação padrão do Portal Padrão?**

Não

**Está utilizando o buildout padrão como o usado em https://github.com/plonegovbr/portal.buildout?**

Não

**Está utilizando o arquivo de versões de pinagens das dependências como definido em https://github.com/plonegovbr/portalpadrao.release e https://github.com/plonegovbr/brasil.gov.portal#requisitos?**

Sim. É uma instalação brasil.gov.portal que aponta para a versão https://github.com/plonegovbr/portalpadrao.release

**Quais são os passos para reproduzir o problema?**

Marcar a opção "Utilizar topo expansível" e utilizar uma imagem de fundo. Para efeito de testes, usamos uma imagem randômica:

![random_25795](https://user-images.githubusercontent.com/8203476/55488385-607b5a00-5606-11e9-9f62-caf7d2d28a63.jpg)

Além disso, também preenchemos os demais campos: "Notícias em destaque", "Serviços em destaque" e "Assuntos em alta".

**Qual é o retorno esperado? O que você vê ao contrário do esperado?**

Um topo expansível, nos moldes de brasil.gov.br:

![Seleção_160](https://user-images.githubusercontent.com/8203476/55488413-712bd000-5606-11e9-834f-61e74bd59c85.png)

Ao invés disso, temos a renderização tradicional:

![Seleção_161](https://user-images.githubusercontent.com/8203476/55489153-c0263500-5607-11e9-8bb1-5a385512fb6c.png)

Além disso, como pode se ver no print, sumiu a busca.

**A aplicação forneceu algum erro no console ou na interface?**

Não.

**Por favor forneça informações adicionais que possam complementar o seu relato:**

Gostaríamos de saber se esse é um erro da aplicação ou se faltou alguma outra configuração.

O manual em http://www.portalpadrao.gov.br/manuais/gestao-de-conteudo/manual-gestao_rev.pdf/view está incorreto pois cita em sua página 140 que esta configuração ficaria acessível via uma opção não disponível no painel de controle: "Configurações
do Cabeçalho". Foi necessário acessar a opção "Portal Padrão".

**Comment by agnogueira** [comment]
@idgserpro  Se não me engano isso só vai aparecer quando tivermos uma capa chamada página inicial definida como visão padrão na raiz do portal. Essa busca expandida só é aplicada na capa inicial, não em outros lugares

**Comment by agnogueira** [comment]
@idgserpro parte desse problema que você citou já está cadastrado aqui: https://github.com/plonegovbr/brasil.gov.temas/issues/204

**Comment by agnogueira** [comment]
@idgserpro Apenas para registrar, aqui funcionou normalmente

**Comment by idgserpro** [comment]
Realmente alterando o nome funciona, mas isso é um problema de usabilidade uma vez que esse objeto não é criado por padrão e não temos essa informação de forma clara.

O manual também está incorreto pois cita em sua página 140 que esta configuração ficaria acessível via uma opção não disponível no painel de controle: "Configurações do Cabeçalho". Foi necessário acessar a opção "Portal Padrão".

**Comment by agnogueira** [comment]
@idgserpro Sim, concordo com a falta de informação. Acho que o maior problema aqui talvez seja não ter a capa criada.

Parece que o manual precisa de uma boa revisão.
